test/controllers/api/v2/docker_images_controller_test.rb

```diff
@@ -4,9 +4,7 @@ module Katello
   class Api::V2::DockerImagesControllerTest < ActionController::TestCase
     def models
       @repo = Repository.find(katello_repositories(:redis))
-      @image = @repo.docker_images.create!({:image_id => "abc123", :uuid => "123xyz"},
-                                           :without_protection => true
-                                          )
+      @image = @repo.docker_images.create!({:image_id => "abc123", :uuid => "123xyz"})
```

Comment:
Why are we removing ":without_protection"? Is that left over from the strong parameters gem?